All project code/views/partials/calendar.ejs

```diff
@@ -4,6 +4,7 @@
         <% let currDate = date.day; %>
         <% let currMonth = date.month; %>
         <% let currYear = date.year; %>
+        <% let currWeek = date.week; %>
```

Comment:
Is removing these extra variables possible? I think having two of the same variables makes things harder to read, and is probably redundant (unless I'm missing something)

Reply:
EJS cannot update in real time. The reason for the separate variables is because one version needs to be incremented when building the calendar, and the other needs to be altered and stored for a call to next or previous. You can't do direct manipulations with date.day etc because those are not local variables, those are request variables

Reply:
Alright, but the frontend isn't modifying the current date, is it? That's where the confusion from the variable names came from, because I thought everything would be dictated server side, except for the viewing date.

The solution works, I guess variable names don't really matter

Reply:
Yes the frontend is incrementing the current date in the for loop. That's how the date increases with each new card